src/main.rs

```diff
@@ -9,30 +9,25 @@ use std::{env, fs};
 fn main() -> Result<()> {
     let config = config()?;
 
-    let input_device = config
```

Comment:
Why did this stuff disappear?

Reply:
It is simplified because `config.audio` is no longer optional, so it doesn't take a bunch of code to pull out the devices from it.